packages/xod-client/src/editor/actions.js

```diff
@@ -31,19 +31,6 @@ import {
 
 import { LINK_ERRORS } from '../messages/constants';
 
-export const setMode = mode => (dispatch, getState) => {
```

Comment:
Oooh, nice.

Reply:
Sound like irony. Do you think that's bad solution?

Reply:
Nope. In #783, I got rig of calls to `setMode`s in other actions(see [here](https://github.com/xodio/xod/pull/783/files#diff-717715e380a7256fcfd3949a5309d803L63) an onwards) to be able to "localize" `mode` in the future.